examples/grover/README.md

```diff
@@ -0,0 +1,123 @@
+# A General Grover Model
+
```

Comment:
Please include a reference link to the code, as we do in all tutorials.

Reply:
Done